### PR TITLE
TW-931: update android logo

### PR DESCRIPTION
**Pull request by imGok**
#931

### demo android
https://github.com/linagora/twake-on-matrix/assets/43041967/8b841840-3c80-4bec-a352-8e85b8d37eab

### Demo ios
https://github.com/linagora/twake-on-matrix/assets/43041967/0d53da8e-d553-455d-a93c-0e04439c0a3f
### demo ios with new splash screen
https://github.com/linagora/twake-on-matrix/assets/43041967/b772a45e-0c52-44a0-93c0-de041f44d8a9

### Demo web
https://github.com/linagora/twake-on-matrix/assets/43041967/a1f97a85-181e-4713-bc47-f7d468d296ee

### CODE REVIEW

File: android/app/src/main/res/drawable-mdpi/notifications_icon.png
Comment (hoangdat) [inline android/app/src/main/res/drawable-mdpi/notifications_icon.png:1]: how you can have this file?

File: android/app/src/main/res/mipmap-hdpi/ic_launcher_round.png
Comment (hoangdat) [inline android/app/src/main/res/mipmap-hdpi/ic_launcher_round.png:1]: I think it must be rounded version, Android studio will help us

File: pubspec.lock [hunk 1]
```diff
@@ -1567,7 +1567,7 @@ packages:
     description:
       path: "."
       ref: "twake-supported-0.22.6"
-      resolved-ref: "ed4ec9267bebd557c414e43224202431d57ea954"
+      resolved-ref: ed4ec9267bebd557c414e43224202431d57ea954
```
Comment (hoangdat) [inline pubspec.lock:1570]: OMG, always see this kind of change? what is the problem?

Comment (sherlockvn) [reply pubspec.lock:1570]: yeah, no need this change

**Review by sherlockvn — APPROVED** [review]

File: android/build/ios/Pods.build/Release-iphonesimulator/DKImagePickerController.build/dgph
Comment (drminh2807) [inline android/build/ios/Pods.build/Release-iphonesimulator/DKImagePickerController.build/dgph:1]: why we have ios build inside android build ?
you should ignore it

File: assets/icons/icon_launcher.png
Comment (hoangdat) [inline assets/icons/icon_launcher.png:1]: this icon use for what? can we use the `rounded` version?

File: assets/icons/notifications_icon.png
Comment (hoangdat) [inline assets/icons/notifications_icon.png:1]: Notification for what?

**Comment by hoangdat** [comment]
srr, but please help me improve the problem too. favicon of Twake so bad.

Screenshot 2023-11-10 at 16 30 39

**Comment by hoangdat** [comment]
- [ ] should comeback to this notification icon (I considered carefully)

image

- [ ] this icon is not cover well in device 

Screenshot 2023-11-10 at 16 34 59

- [ ] must be a black background in splash screen
- [ ] and demo for android < 30

File: assets/icons/icon_launcher.png
Comment (sherlockvn) [reply assets/icons/icon_launcher.png:1]: removed

File: assets/icons/notifications_icon.png
Comment (sherlockvn) [reply assets/icons/notifications_icon.png:1]: removed

**Comment by sherlockvn** [comment]
> srr, but please help me improve the problem too. favicon of Twake so bad.
> 
> Screenshot 2023-11-10 at 16 30 39

image
DOne

**Comment by sherlockvn** [comment]
> * [x]  should comeback to this notification icon (I considered carefully)
> 
> image
> 
> * [ ]  this icon is not cover well in device
> 
> Screenshot 2023-11-10 at 16 34 59
> * [x]  must be a black background in splash screen
> * [x]  and demo for android < 30
> * [x]  footer of iOS splash screen is too low, too near the bottom
> 
>  RPReplay_Final1699642406.MP4

- this icon is not cover well in device
Here is the gmail icon, I think it's the best solution
image

**Comment by sherlockvn** [comment]
Demo in android 29

https://github.com/linagora/twake-on-matrix/assets/43041967/fd978121-4bd4-46f3-be27-db35fd84d3c0

Demo again in again 33

https://github.com/linagora/twake-on-matrix/assets/43041967/2b2a620b-52b2-40b5-bc4d-db50300702c9

**Review by drminh2807 — APPROVED** [review]

**Review by nqhhdev — APPROVED** [review]

**Review by hoangdat — APPROVED** [review]

**Pull request merged**